pints/toy/_constant_model.py

```diff
@@ -106,8 +106,6 @@ def simulateS1(self, parameters, times):
             #   [df2/dp1, df2/dp2]]
             # i.e.
             #  [[1, 0],
-            #   [0, 2]]
-            # i.e. np.diag(self._r)
-            dy = np.tile(np.diag(self._r), (len(times), 1, 1))
+            #   [0, 1]]
+            dy = np.tile(np.diag(np.ones(len(self._r))), (len(times), 1, 1))
```

Comment:
Thanks Ben!
Could just be `self._n` instead of `len(self._r)` then I think?

Reply:
Main docstring needs updating too!

Reply:
@MichaelClerx Good idea. Both changed in my latest commit!